package-versions/setup.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: Setup
+parent: Package Versions
+nav_order: 1
+---
+
+# Setup and Configure Central Package Version Management
+
+## Create the Central Package Version Management File
+In the solution directory, create a `Directory.Packages.props` file to define versions for each package used in your projects.
+
+#### Directory.Packages.props
+```xml
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <ItemGroup>
+        <PackageVersion Include="foo" Version="1.0.0" />
+    </ItemGroup>
+</Project>
+```
+
+With a `<PackageVersion>` item within an `<ItemGroup>`, the `Version=""` will be used wherever the specified package is called by a `<PackageReference>`.
+
+`<PackageVersion>` is an MSBuild item that allows a project to specify a version for a NuGet package without performing the import.
+
+## Enable Central Package Version Management
+Once a `Directory.Packages.props` file has been created, add a `<ManagePackageVersionsCentrally>` item set to `true` in the `<PropertyGroup>` in a `Directory.Build.props` file.
```

Comment:
We do not need `Directory.Build.props`. We would like this to be in `Directory.Packages.props`

Reply:
The best, in my opinion, would be to automatically enable `ManagePackageVersionsCentrally` if `Directory.packages.props` is present.

Reply:
I also think that this should automatically be enabled once a `Directory.packages.props`-file exists.